content/integrations/suzaku.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: Suzaku
+category: Developer Tooling
```

Comment:
Wondering if Suzaku should go under `Developer Tooling` or if we should create a new category. Maybe Ava Labs will tell us